src/card.tsx

```diff
@@ -424,10 +424,12 @@ const CardContent = ({
                             </div>
                         )}
                         {headline && (
-                            // assuming that the headline will always be followed by one of: pretitle, title, subtitle, description
```

Comment:
All cards were assuming this before, except of AdvancedDataCard. In https://github.com/Telefonica/mistica-web/pull/1292, I've done the same for the AdvancedDataCard, and it broke one screenshot test in webapp (they use pretitle and extra prop to render everything else).

I've removed all these unnecesary assumptions from everywhere